Model/Button/GetCartButtonConfig.php

```diff
@@ -274,9 +274,21 @@ private function getOrderContent($quote): array
                 'quantity' => $item->getQty(),
                 'total' => (float)$item->getRowTotalInclTax(),
                 'url' => $item->getProduct()->getProductUrl(),
-                'image' => $this->storeManager->getStore()->getBaseUrl(UrlInterface::URL_TYPE_MEDIA) . 'catalog/product' . $item->getProduct()->getImage(),
+                'image' => $this->getProductImageUrl($item->getProduct())
             ];
         }
         return $orderContent;
     }
+
+    /**
+     * @param $product
+     * @return string
+     */
+    private function getProductImageUrl($product)
+    {
+        if (!$product->getData('small_image')) {
```

Comment:
If we are using the image roles, maybe we can add an additional fallback to the "thumbnail" role so we have an additional option?

Reply:
I'll try this